### DISCUSSION TITLE
Use cases for curb metrics

### DISCUSSION THREAD

**Discussion by schnuerle** [openmobilityfoundation/curb-data-specification#28] [Data Specification]
What are your data use cases for your curb management program?  What are you trying to measure and improve with data?  

Use cases are the starting point for data collection and by defining and collecting your use cases here, we can work toward the next step which is [defining a methodology to calculate raw numbers](https://github.com/openmobilityfoundation/curb-data-specification/discussions/27) needed to meet your use cases.

**Comment by bhamlinSDOT** [comment]
Examples of what measurements we request (or would like to request) in order to make improvements/changes in Seattle:

**Measurement -> Improvement**
- Measure double parking or parking in center left turn lane
  - Increase the amount/length/availability of load zones
- Time of day for usage
  - Adjust regulations to meet delivery needs and free up space for other activities 
- Measure dwell time
  - Are our time limits appropriate?
  - Do they need to be shorter or longer?
    - Shorter to increase turnover
    - Longer to allow for activity to to occur without a citation
  - Are pricing changes necessary?
- Latent demand (how many vehicles would like to park but cannot)
  - Increase availability of desired zone
- What type of vehicles/activities are using the zones? (compliance check)
  - More targeted enforcement
  - Make space for other uses
  - Improve signage

**Comment by schnuerle** [comment]
We also have our [Curbs Use Case document](https://github.com/openmobilityfoundation/curb-data-specification/wiki/CDS-Curbs-Use-Cases) that focuses on 1 of 3 of the CDS APIs: Curbs. There is some overlap with Metrics. We will use the comments here to help create and fill in our Metrics use case doc.

**Comment by tomcarr178** [comment]
Examples of what LA City would like to measure:
Curb policy: Metered vs. free, type of zone, time limits, restrictions, etc.
Dwell time and occupancy: not only the duration but the arrival and departure time.  
Type of vehicle:  If possible, car, truck, taxi, TNC, etc. and if we can distinguish ZEV vs. gas
Paid/unpaid parking: Rate, payment data. Be able to identify disabled placard use vs. violation for unpaid parking.

**Comment by tomcarr178** [comment]
LA City use cases:
LA Express Park: Measure occupancy, duration of occupancy, and payment
ZE Delivery Zones: Measure occupancy, duration of occupancy, type of vehicle that occupies the space
